# This file is for general simulator params
rate:
    simulator:
        # The rate at which the simulator bridge runs, this must be faster
        #  than any rates used for topics published by the simulator
        #  such as control, depth, sensor in cobalt_sim.yaml and
        #  any rate parameters below
        simulator_bridge: 30
        position: 10
        euler: 20
        obstacle_pos: 1
        hydrophone_deltas: 0.5
        lin_accel: 20

simulator:
    visualize_thrusters: true
    visualizer_update_rate: 10
    ports:
        simulated_thruster: "/dev/null"
        simulated_sensor: "/dev/null"
    significant_links:
        -   name: "gate_side_a::gate_right"
            label: "start_gate_post"
            label_id: 1
            height: 1.5
            width: 0.1
            depth: 0.1
        -   name: "gate_side_a::gate_left"
            label: "start_gate_post"
            label_id: 1
            height: 1.5
            width: 0.1
            depth: 0.1
        -   name: "buoys_side_a::red_buoy"
            label: "red_buoy"
            label_id: 2
            height: 0.2
            width: 0.2
            depth: 0.2
        -   name: "buoys_side_a::green_buoy"
            label: "green_buoy"
            label_id: 3
            height: 0.2
            width: 0.2
            depth: 0.2
        -   name: "buoys_side_a::yellow_buoy"
            label: "yellow_buoy"
            label_id: 4
            height: 0.2
            width: 0.2
            depth: 0.2
        -   name: "channel_side_a::bottom"
            label: "nav_channel_bar"
            label_id: 5
            height: 2.0
            width: 0.1
            depth: 0.1
        -   name: "channel_side_a::left"
            label: "nav_channel_post"
            label_id: 6
            height: 1.0
            width: 0.1
            depth: 0.1
        -   name: "channel_side_a::right"
            label: "nav_channel_post"
            label_id: 6
            height: 1.0
            width: 0.1
            depth: 0.1
        -   name: "path_marker2_side_a::top"
            label: "path_marker"
            label_id: 7
            height: 0.1
            width: 0.15
            depth: 1.2
        -   name: "path_marker1_side_a::top"
            label: "path_marker"
            label_id: 7
            height: 0.1
            width: 0.15
            depth: 1.2
